src/sass/_empty_state.scss

```diff
@@ -16,6 +10,32 @@
     width: 2.5rem;
   }
 
+  &.full-width {
+    width: 100%;
+  }
+
+  &.show-left-icon {
```

Comment:
I think we should align the empty states to the layout as we have it (see below). So wouldn't need this extra classes for `show-left-icon` and `full-width`.

![image](https://github.com/canonical/lxd-ui/assets/1155472/aa3a0ed5-9978-430d-bb30-696a439adccd)

Reply:
The empty states on the main permission pages are consistent with other pages. The left icon structure is for empty state inside the panels, it's a new design so I don't think we should change it since it will be implemented in other projects as well.

![Screenshot from 2024-04-26 17-13-33](https://github.com/canonical/lxd-ui/assets/151511568/0307068a-f21d-49fa-9d64-79a43a0f9055)

Reply:
We will use existing empty state for now, noting that this design may become the new empty state for all apps.